tests/filesystem.go

```diff
@@ -319,17 +325,20 @@ func formatPartition(ctx context.Context, partition *types.Partition) error {
 	return nil
 }
 
-func writePartitionData(device string, contents string) error {
+func writePartitionData(device string, contents string) (err error) {
 	bzipped, err := base64.StdEncoding.DecodeString(contents)
 	if err != nil {
 		return err
 	}
 	reader := bzip2.NewReader(bytes.NewBuffer(bzipped))
-	f, err := os.OpenFile(device, os.O_WRONLY, 0644)
+	var f *os.File
+	f, err = os.OpenFile(device, os.O_WRONLY, 0644)
```

Comment:
hmm, is there a lint that told us we needed to not use ':-=' and explicitly use var?

Reply:
Not really, it's an unrelated change. I'll split it into another commit.